data.js

```diff
@@ -358,7 +358,28 @@ class Data {
         amount: -shares,
       });
       return true;
+
+  } if (shares==0) {
```

Comment:
what is this block of code for?

Reply:
Oh oops, I think that was from when I was going to add a "Max" button (so the user wouldn't be inputting shares), but we scrapped that idea and I forgot about this block of code